presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -293,6 +293,8 @@ public void testExplainAnalyze()
     public void testExplainAnalyzeVerbose()
     {
         assertExplainAnalyze("EXPLAIN ANALYZE VERBOSE SELECT * FROM orders");
+        assertExplainAnalyze("EXPLAIN ANALYZE VERBOSE SELECT rank() OVER (PARTITION BY orderkey ORDER BY clerk DESC) FROM orders");
+        assertExplainAnalyze("EXPLAIN ANALYZE VERBOSE SELECT rank() OVER (PARTITION BY orderkey ORDER BY clerk DESC) FROM orders WHERE orderkey < 0");
```

Comment:
Is there a point in executing these for many different connectors?

Reply:
Concerning this specific issue - no I don't think so.
Yet generally the explain output can be different for different connectors, and potentially we can have connector dependent failures here.
What do you propose?

Reply:
I think it's good to have _some_ run of `EXPLAIN ANALYZE VERBOSE` for all connectors. And this should include single-table selects and eg a  join.

For the actual problem, I would propose introduction of `TestExplain` test class. (Just like `TestGrouping` but with distributed query runner)

Reply:
Not that easy as it looks like. It would make sense to migrate the most of explain tests from `ATQ` to that new class. Yet that is not trivial because the machinery for testing explains lives in  `AbstractTestQueryFramework` superclass. This refactor is too much of an effort IMO for this issue. I will leave it out.